test/unit/dispatch.js

```diff
@@ -62,3 +62,24 @@ test('remote', t => {
         .then(() => runServiceTest('RemoteDispatchTest', t), e => t.fail(e))
         .then(() => dispatch._remoteCall(worker, 'dispatch', 'terminate'), e => t.fail(e));
 });
+
+test('local, sync', t => {
+    dispatch.setServiceSync('SyncDispatchTest', new DispatchTestService());
+
+    const a = dispatch.callSync('SyncDispatchTest', 'returnFortyTwo');
+    t.equal(a, 42);
+
+    const b = dispatch.callSync('SyncDispatchTest', 'doubleArgument', 9);
+    t.equal(b, 18);
+
+    const c = dispatch.callSync('SyncDispatchTest', 'doubleArgument', 123);
+    t.equal(c, 246);
+
+    try {
+        dispatch.callSync('SyncDispatchTest', 'throwException');
```

Comment:
My understanding is that you should be able to use, e.g. 
```suggestion
        t.throws(() => dispatch.callSync('SyncDispatchTest', 'throwException'), new Error('This is a test exception thrown by DispatchTest'));
```

Reply:
Done.